ccloud/ccloud-generate-cp-configs.sh

```diff
@@ -272,6 +272,8 @@ while read -r line
     fi
   fi
 done < "$CONFIG_FILE"
+#max.message.bytes is enforced to 8Mb in CCloud
```

Comment:
For clarification:

1. Is it enforced or recommended?
2. What's the default in CCloud?
3. What's the default in AK?